test/test_jit.py

```diff
@@ -788,6 +788,13 @@ def foo(x):
         x = torch.rand(3, 4)
         self.assertEqual(scripted(x), foo(x))
 
+    def test_partial(self):
+        def sum_fun(x: torch.Tensor, dim: int = 0) -> torch.Tensor:
+            return torch.sum(x, dim=dim)
+        scripted_partial = torch.jit.script(functools.partial(sum_fun, dim=-1))
+        x = torch.ones(5)
+        self.assertEqual(scripted_partial(x), sum_fun(x))
```

Comment:
This seems a bit off to me. `partial` basically alters the default value of `dim`, but it is not reflected in scripted function.